### PR TITLE
Buckle Up: Shuttles now punish you for not riding safely, again!

### PR DESCRIPTION
**Pull request by Iamgoofball**
Not sure why this got reverted back down to 0, but you really gotta use those seat-belts.
:cl: Iamgoofball
tweak: Central Command would like to advise passengers to please wear their seatbelts at all times.
/:cl:

### CODE REVIEW

**Comment by HomocidlePancakes** [comment]
Was moved to emagging shuttle. You’ll have to change that too.

**Comment by Mickyan** [comment]
#41999
Are you going to remap all shuttles to make this work

**Comment by spessbandit** [comment]
buff to mime chair

File: code/modules/shuttle/shuttle.dm [hunk 1]
```diff
@@ -283,7 +283,7 @@
 	var/launch_status = NOLAUNCH
 
 	///Whether or not you want your ship to knock people down, and also whether it will throw them several tiles upon launching.
-	var/list/movement_force = list("KNOCKDOWN" = 3, "THROW" = 0)
+	var/list/movement_force = list("KNOCKDOWN" = 3, "THROW" = 14)
```
Comment (kriskog) [inline code/modules/shuttle/shuttle.dm]: 14? Get out.

Comment (Shadowflame909) [reply code/modules/shuttle/shuttle.dm]: Weee being thrown at hyper speed

**Comment by Ratvar** [comment]
Dude with a wrench should not be killing entire shuttle, someone get goofball away from code

File: code/modules/shuttle/shuttle.dm (hunk 1 above)
Comment (Iamgoofball) [reply code/modules/shuttle/shuttle.dm]: The shuttle literally takes off at light speed. If I wanted to be realistic about it, you would immediately be gibbed by the velocity and mass of the shuttle colliding with you. This works.

**Comment by Iamgoofball** [comment]
@Mickyan I don't think that's an issue. Quite frankly, it's funny.

**Review by 81Denton — CHANGES_REQUESTED** [review]
You just need to fix all shuttles since #41999 removed this because our shuttles aren't designed for stuff being thrown around 24/7

**Comment by Iamgoofball** [comment]
Things going flying because shuttles throw things that aren't anchored down is intentional. Like, that's the point of the PR.

**Comment by 81Denton** [comment]
They also weren't mapped with the idea of throwing items in mind, else there wouldn't be 50 loose items in every shuttle that do fun things like damaging players that spawned in the arrivals shuttle.

**Comment by Iamgoofball** [comment]
This change was made exclusively to the emergency shuttle code

**Comment by Iamgoofball** [comment]
oh wait i edited the wrong file

**Comment by Iamgoofball** [comment]
yeah gimme a bit I'll fix that

**Comment by 81Denton** [comment]
Ahh alright, if it's just the emergency shuttle that's bueno

**Comment by Iamgoofball** [comment]
ya throwin crap around on other shuttles is a no-go, this is only for the emergency shuttle

**Comment by optimumtact** [comment]
This isn't going to work

**Pull request closed without merging**